SwiftPkgsExample/JustPassMeExample/Auth/SignInPassKeysButton.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  SignInPassKeysButton.swift
+//  JustPassMeExample
+//
+//  Created by Sameh Galal on 10/25/23.
+//
+
+import SwiftUI
+import LocalAuthentication
+
+struct SignInPassKeysButton: View {
+  
+  let action: () -> Void
+
+  private var biometryType: LABiometryType {
+    let context = LAContext()
+    var error: NSError?
+    if context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: &error) {
+      return context.biometryType
+    } else {
+      return .none
+    }
+  }
+  
+  init(action: @escaping () -> Void) {
+    self.action = action
+  }
+    var body: some View {
+      Button(action: action) {
+        Image(systemName: self.biometryType == .faceID
+              ? "faceid"
+              : "touchid")
```

Comment:
i think it will better to move this to computed property will make it more readable  what's ur opinion 
`self.biometryType == .faceID
               ? "faceid"
               : "touchid"`